tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -591,24 +591,97 @@ test.describe( 'Set up accounts', () => {
 				).toBeEnabled();
 			} );
 
-			test( 'should display the correct Google Ads ID when connected', async () => {
-				const googleAccountCard =
-					setUpAccountsPage.getGoogleAccountCard();
+			test( 'should send an API request to connect existing Google Ads account', async () => {
```

Comment:
This teas case doesn't assert if there is an API request is sent or even assert anything else. It would be better to add the corresponding assertion.

Reply:
Added assertion.